package.json

```diff
@@ -24,7 +24,9 @@
     "react-redux": "^6.0.1",
     "react-scripts": "^2.1.5",
     "react-select": "^2.4.2",
```

Comment:
I'm pretty sure I use both of these; I at least use the css from the react-select, and its too much of a pain to want to see if I could do it through react-select-2.